input/pagecontent/changes.md

```diff
@@ -180,6 +180,7 @@ This change log documents the significant updates and resolutions implemented fr
   - corrected the Observation combined search parameter 'patient+category+status' from SHALL to SHOULD [FHIR-45390](https://jira.hl7.org/browse/FHIR-45390)
   - corrected narrative of Location identifier parameter requester requirements on providing both the system and code values from SHOULD to MAY [FHIR-46209](https://jira.hl7.org/browse/FHIR-46209)
   - replaced security and authorization requirements with pointer to Security and Privacy page [FHIR-46067](https://jira.hl7.org/browse/FHIR-46067)
+  - added entries for referenced resources, DocumentReference and RelatedPerson, in line with the Must Support requirements in AU Core profiles [FHIR-46546](https://jira.hl7.org/browse/FHIR-46546)
```

Comment:
rephrase to just "added missing entry for DocumentReference and RelatedPerson resources."
this text "in line with ..." is like gravity - it applies to all changes ever.

Reply:
Amended, thanks

Reply:
spelling issues. entry should be plural.

Reply:
Amended, thanks